Sources/OutlineView/OutlineView.swift

```diff
@@ -47,6 +47,102 @@ where Drop.DataElement == Data.Element {
     var dragDataSource: DragSourceWriter<Data.Element>?
     var dropReceiver: Drop
 
+    // MARK: Initializers without SeparatorInsets (MacOS 10.15+)
```

Comment:
I think it's perfectly acceptable to break out the inits into extensions for macOS 10.15 and macOS 11+.

This gives you the added benefit of being able to annotate the entire extension with `@available(macOS 11, *)` instead of an individual init.

Reply:
Agreed. So I just reorganized the initializers here: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/c521abc986de302c3a5ff8416ddec73062e70e1e

Hopefully that looks good to you, too :-)

Reply:
Looks great!